src/test/java/com/puppycrawl/tools/checkstyle/api/BeforeExecutionFileFilterSetTest.java

```diff
@@ -100,4 +101,13 @@ public void testClear() {
                 .isEmpty();
     }
 
+    @Test
+    public void testUnmodifiableSet() {
```

Comment:
please make brief comment above to explain why we can not do Input validation and what we try to archive

something like: `Causing low level configuration setup problem. Required to cover pitest mutation.`

Reply:
@romani Done. Please check message.

Reply:
Concert to multi line comment

Reply:
@romani  Done.

Reply:
@romani 
Honestly, I didn't understand the reasoning as to why we are not writing checks with input files. Is it because no matter which check we write, we won't be able to modify the values of these tokens which we otherwise can modify through pure JUnit tests?

I found some similar comment below:
https://github.com/checkstyle/checkstyle/blob/d4ddf4b7d1c421c2b3ff11532832fa4e3a989dde/src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java#L393-L403